iceoryx_posh/source/popo/ports/interface_port_data.cpp

```diff
@@ -23,7 +23,7 @@ namespace popo
 {
 InterfacePortData::InterfacePortData(const RuntimeName_t& runtimeName, const capro::Interfaces interface) noexcept
     : BasePortData(capro::ServiceDescription(
-                       capro::InvalidIDString, capro::InvalidIDString, capro::InvalidIDString, {0, 0, 0, 0}, interface),
+                       capro::InvalidString, capro::InvalidString, capro::InvalidString, {0, 0, 0, 0}, interface),
                    runtimeName,
```

Comment:
The default ctor should take care or this

Reply:
You mean a c'tor just like `capro::ServiceDescription(Interfaces interfaceSource)`? Seems like `InterfacePort`'s don't need the `ServiceDescription` of the `BasePort`. If you're ok with it, I would like to keep it like that for now as the `InterfacePort`'s will be removed anyway as part of #415.

Reply:
No, I mean a `ServiceDescription foo;` should set the members to the `capro::InvalidString`, etc.

Then this would become `BasePortData(ServiceDescription(), runtimeName, ...` or even `BasePortData({}, runtimeName, ...`

Reply:
oh, I overlooked the `interface` parameter since it is hidden due to (insert some swearing regarding this artificial UI limitation to 800px or whatever even it there is so much space left and right) github UI